initfiles/bash/etc/systemd/system/CMakeLists.txt

```diff
@@ -37,18 +37,24 @@ set(dfuserver "dfuserver" "--daemon %i" "dafilesrv.service")
 set(eclagent "agentexec" "--daemon %i" "dafilesrv.service")
 set(eclccserver "eclccserver" "--daemon %i" "dafilesrv.service")
 set(eclscheduler "eclscheduler" "--daemon %i" "dafilesrv.service")
-set(esp "esp" "--daemon %i" "dafilesrv.service")
```

Comment:
Why was this line deleted? Seems unrelated (and may explain the smoketest failure?)

Reply:
This line was deleted because there is now an esp@instance.service file, and we don't generate it from the component@instance.service template.